ios-xcframework/Podfile.lock

```diff
@@ -674,6 +674,6 @@ SPEC CHECKSUMS:
   WordPress-Aztec-iOS: 7d11d598f14c82c727c08b56bd35fbeb7dafb504
   Yoga: f7decafdc5e8c125e6fa0da38a687e35238420fa
 
-PODFILE CHECKSUM: 1caec69a3de9c6e37113c8eb5dbdb2d2b52c5108
+PODFILE CHECKSUM: 8f2b84cf5105963b4ef56ad18d6d64ab24334789
```

Comment:
Notice the only change is in the checksum. This mean that the resolved dependencies with this new approach are exactly the same as before 🎉 

The different checksum is due to the text content of `Podfile` having changed.